python/sensitive_detectors.py

```diff
@@ -65,11 +65,20 @@ class HcalSD(simcfg.SensitiveDetector) :
     """SD for the HCal
 
     Separate from the other calorimeters since it includes a Birks law
-    estimate. No other parameters
-    
+    estimate.
+
+    Parameters
+    ----------
+    gdml_identifiers : list[str]
+
+        A list of strings used to determine which volumes in the Hcal are
+        considered sensitive. Any volume name containing at least one of these
+        identifiers will have a sensitive detector attached.
+
     """
-    def __init__(self) :
+    def __init__(self, gdml_identifiers = ['ScintBox', 'scint_box']) :
```

Comment:
Do these defaults cover _both_ the prototype detector and the mainline detector?

If so, I'd just add a comment saying that so people aren't confused. My first thought was that there could be two volumes in the same detector that match either (or both?) of these strings.

Reply:
That's a good catch. These are the defaults for the mainline (ScintBox) and prototype (scint_box). The check in isSensDet will apply if either of these are a substring match to the volume name.